charts/zeppelin-with-spark/README.md

```diff
@@ -68,7 +68,7 @@ Use the URL printed by the above command to access Zeppelin.
 
 NOTE: Below command will work ONLY after a Spark job is launched. Run a Spark Notebook, first. 
 ```
-   export SPARK_UI_SERVICE_IP=$(kubectl get svc --namespace default example-zeppelin-spark-web-ui -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+   export SPARK_UI_SERVICE_IP=$(kubectl get svc --namespace default example-zeppelin -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
    echo http://$SPARK_UI_SERVICE_IP:4040
 ```
```

Comment:
Instead of 2 different commands to get Zeppelin URL and Spark URL, maybe combine this into one like you have done for NOTES.txt. Also the above command uses 'example-zeppelin' instead of 'example-zeppelin-with-spark'

Reply:
Done. Correct service name is 'example-zeppelin'.